plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -609,19 +560,15 @@ module.exports = {
       }
     }
 
-    if (field === 'healthChecks') {
-      if (newState[index].healthChecks == null) {
-        newState[index].healthChecks = [];
-      }
-
-      if (this.healthChecks[index] == null) {
-        this.healthChecks[index] = {};
+    if (field === 'healthCheck') {
+      if (this.healthCheckState[index] == null) {
+        this.healthCheckState[index] = {};
       }
 
-      newState[index].healthChecks = MultiContainerHealthChecks.call(
-        this.healthChecks[index],
-        newState[index].healthChecks,
-        {type, path: path.slice(2), value}
+      newState[index].healthCheck = multiContainerHealthFormReducer.call(
+        this.healthCheckState[index],
+        newState[index].healthCheck || {},
```

Comment:
Let's change `multiContainerHealthFormReducer` to handle undefined healthChecks instead, of defaulting the health check to an empty object here.